docs/track-record-sections.md

```diff
@@ -1,9 +1,11 @@
 # Track Record Sections
-The `/track-record` page is managed by the file `content/data/track-record-sections.yaml`.
+This is now hard coded in to the HTML to give more flexibility
```

Comment:
Feels a bit like a step back/loss. But ya maybe its better to be able to each custom rather than try to standardise them.